public/app/features/browse-dashboards/components/FolderActionsButton.tsx

```diff
@@ -100,10 +101,12 @@ export function FolderActionsButton({ folder }: Props) {
   return (
     <>
       <Dropdown overlay={menu} onVisibleChange={setIsOpen}>
-        <Button variant="secondary">
-          Folder actions
-          <Icon name={isOpen ? 'angle-up' : 'angle-down'} />
-        </Button>
+        <Trans i18nKey="browse-dashboards.folder-actions-button.folder-actions">
+          <Button variant="secondary">
+            Folder actions
+            <Icon name={isOpen ? 'angle-up' : 'angle-down'} />
+          </Button>
+        </Trans>
```

Comment:
We don't want to give translators ability to change how these components are laid out, so we should move `<Trans />` to inside the `<Button />`

```ts
<Button variant="secondary">
  <Trans i18nKey="browse-dashboards.folder-actions-button.folder-actions">
    Folder actions
  </Trans>
  <Icon name={isOpen ? 'angle-up' : 'angle-down'} />
</Button>
```

Reply:
Done :)